org.eclipse.dawnsci.analysis.dataset/src/org/eclipse/dawnsci/analysis/dataset/impl/Image.java

```diff
@@ -51,11 +51,11 @@
 	private static IImageFilterService filterService;
 	private static IImageTransform transformService;
 
-	public static void setImageFilterService(IImageFilterService ifs) {
+	public void setImageFilterService(IImageFilterService ifs) {
 		filterService = ifs;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2696)